src/main/java/com/academia/library/model/Order.java

```diff
@@ -36,10 +41,10 @@
     private Long id;
 
     @Column
-    private BigDecimal amount;
+    private Boolean deleted = Boolean.FALSE;
```

Comment:
Please delegate default value to liquibase or set it manually. It looks implicit a little

Reply:
Ok, i fix this moment in mapstruct